src/Apps/IdentityVerification/__tests__/IdentityVerificationApp.jest.tsx

```diff
@@ -1,32 +1,37 @@
-import { IdentityVerificationAppTestQueryRawResponse } from "__generated__/IdentityVerificationAppTestQuery.graphql"
-import deepMerge from "deepmerge"
-import { createTestEnv } from "DevTools/createTestEnv"
 import { graphql } from "react-relay"
-import { IdentityVerificationAppQueryResponseFixture } from "../__fixtures__/routes_IdentityVerificationAppQuery"
 import { IdentityVerificationAppFragmentContainer } from "../IdentityVerificationApp"
 import { IdentityVerificationAppTestPage } from "./Utils/IdentityVerificationAppTestPage"
 import { mockLocation } from "DevTools/mockLocation"
 import { HttpError } from "found"
 import { Toasts, ToastsProvider } from "@artsy/palette"
+import { setupTestWrapper } from "DevTools/setupTestWrapper"
+import { MockBoot } from "DevTools"
+import { useTracking } from "react-tracking"
+import { createMockEnvironment, MockPayloadGenerator } from "relay-test-utils"
 
 jest.unmock("react-relay")
-jest.unmock("react-tracking")
+jest.mock("react-tracking")
 jest.mock("Utils/Events", () => ({
   postEvent: jest.fn(),
 }))
 jest.mock("found")
 
-const mockPostEvent = require("Utils/Events").postEvent as jest.Mock
-
-const setupTestEnv = () => {
-  return createTestEnv({
-    TestPage: IdentityVerificationAppTestPage,
-    Component: props => (
-      <ToastsProvider>
-        <Toasts />
-        <IdentityVerificationAppFragmentContainer {...props} />
-      </ToastsProvider>
-    ),
+describe("IdentityVerification route", () => {
+  let sendProps
+  let trackEvent
+
+  const { getWrapper } = setupTestWrapper({
+    Component: (props: any) => {
+      const propsFiltered = sendProps ? props : {}
```

Comment:
this is a bit hackish but as far as I tested our current relay setup is not allowing to mock resolver types as `null/undefined` 🤷